google/earthenterprise
# I1202 Can't select .KIP as source

Issue: https://github.com/google/earthenterprise/issues/1202 

Steps to test:
1. Open Fusion
2. Navigate to Open Image source
3. Navigate to .KIP project, and open directory (.KIP) and select "header.xml";  click "open".
4. Will see .KIP in resource listing, *without* the "header.xml" portion of the path.
